samza-sql/src/test/java/org/apache/samza/sql/planner/TestSamzaSqlValidator.java

```diff
@@ -59,6 +59,42 @@ public void testBasicValidation() throws SamzaSqlValidatorException {
     new SamzaSqlValidator(samzaConfig).validate(sqlStmts);
   }
 
+  @Test
+  public void testRepeatedTwiceFieldsValidation() throws SamzaSqlValidatorException {
+    Map<String, String> config = SamzaSqlTestConfig.fetchStaticConfigsWithFactories(1);
+    config.put(SamzaSqlApplicationConfig.CFG_SQL_STMT,
+        "Insert into testavro.outputTopic select id, true as bool_value, false as bool_value"
```

Comment:
I think we need to add comments here on why we support a field repeated twice and not 3 times?